batcher/aligned/src/main.rs

```diff
@@ -464,12 +479,17 @@ async fn main() -> Result<(), AlignedError> {
                 })?;
 
             let keystore_path = &deposit_to_batcher_args.keystore_path;
+            let private_key = &deposit_to_batcher_args.private_key;
```

Comment:
add here only 1 of these can be set, to avoid user setting conflicting private key AND keystore.